core/errors/packager.h

```diff
@@ -0,0 +1,19 @@
+#ifndef SORBET_CORE_ERRORS_PACKAGER_H
+#define SORBET_CORE_ERRORS_PACKAGER_H
+#include "core/Error.h"
+
+namespace sorbet::core::errors::Packager {
+constexpr ErrorClass MustBeTypedStrict{8001, StrictLevel::False};
```

Comment:
These numbers are out of order w.r.t. the phases of Sorbet. Starting at 8000 makes me thing that it comes after infer. What do you think about doing something similar to `core/errors/rewriter.h`, which starts at 3500? Since this comes after rewriter and before namer, maybe you could do something like 3700 or 3800?

Reply:
Works for me! I didn't know the convention, and chose an unallocated namespace.

Reply:
Switched to 3700 namespace.